module/mempool/stdmap/identifier_map.go

```diff
@@ -70,18 +58,13 @@ func (i *IdentifierMap) Append(key, id flow.Identifier) error {
 // Otherwise it returns nil and false.
 func (i *IdentifierMap) Get(key flow.Identifier) ([]flow.Identifier, bool) {
```

Comment:
```suggestion
func (i *IdentifierMap) Get(key flow.Identifier) (flow.IdentifierList, bool) {
```

In other places as well.